src/atom-environment.coffee

```diff
@@ -183,6 +187,8 @@ class AtomEnvironment extends Model
     })
     @themes.workspace = @workspace
 
+    @autoUpdater = new AutoUpdateManager({@applicationDelegate})
```

Comment:
I notice you aren't calling `dispose` (which I suggested renaming to `destroy` in another comment). Should you do that in `AtomEnvironment.prototype.destroy`?